code/TagField.php

```diff
@@ -184,20 +159,37 @@ protected function getSuggestURL() {
 	protected function getOptions() {
 		$options = ArrayList::create();
 
+		/**
```

Comment:
Still not happy with the block level comments inside a method body (I think Damian agreed last time I've brought this up). Does your IDE ignore the hint if its defined through a `//` comment?